internal/schema/connection.go

```diff
@@ -119,6 +119,14 @@ func GetKbFWConnectionBlock() fwschema.Block {
 		MarkdownDescription: "Kibana connection configuration block.",
 		NestedObject: fwschema.NestedBlockObject{
 			Attributes: map[string]fwschema.Attribute{
+				"api_key": fwschema.StringAttribute{
+					MarkdownDescription: "API Key to use for authentication to Kibana",
+					Optional:            true,
+					Sensitive:           true,
+					Validators: []validator.String{
+						stringvalidator.ConflictsWith(usernamePath, passwordPath),
```

Comment:
I wonder if it makes sense to have this on the username/password fields for consistency. WDYT?

Reply:
Consistency with ES? Currently this is the same as the ES FW connection block.